backend/src/calibration.py

```diff
@@ -82,8 +85,8 @@ def _gen_siglist(packet):
     cols, rows = target.get_dims()
 
     siglist = []
-    for r in range(0, rows):
-        for c in range(0, cols):
+    for c in range(0, cols):
+        for r in range(0, rows):
```

Comment:
Confirmed this doesn't quite fix calibration right?

Reply:
actually I see your TODO above, does this not deal with the orientation issue?

Reply:
It does not fix it yet. I just wanted to put the changes from the meeting in. This does not affect any output since the entire thing is still broken. Its just a stub of the eventual fix.